canonical/ubuntu.com
# WD-12056 - rebrand /cloud

## Done

Rebrand /cloud No content was changed, layout only. 

## QA

- [demo link](https://ubuntu-com-14083.demos.haus/cloud)
- [figma](https://www.figma.com/design/NpI2wzW86XXPWxsSeomYfy/u.c%2Fcloud?node-id=0-1&t=GiTLYjpmq1sNNmhM-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12056](https://warthogs.atlassian.net/browse/WD-12056)

[WD-12056]: https://warthogs.atlassian.net/browse/WD-12056?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
